content/php/concepts/string-functions/terms/ord/ord.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'ord()'
+Description: 'Returns an integer between 0 and 255 corresponding to the binary value of the first character of a string.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`ord()`** function takes a string as its only parameter and returns an integer between 0 and 255 based on the first character in that string.
+
+The integer returned is the decimal equivalent of the binary representation of the character. This binary representation conforms to the [ASCII standard](https://www.ascii-code.com/). Therefore, the integer returned is the ASCII decimal value of the first character in the parameter string.
+
+## Syntax
+
+```pseudo
+ord(string)
+```
+
+Required Parameters:
+
+-`string` is the string that the function will take the first character of
+
+## Example
+
+```php
+<?php
+echo ord("equanimity");
+?>
+```
+
+This results in the output:
+
+```shell
+101
+```
+
+In the above example, the first character of the string "equanimity" is "e". In accordance with the ASCII standard, this is represented in binary as 110 0101. This binary value is equivalent to the decimal value 101. Therefore the call to the `ord()` returns 101.
```

Comment:
```suggestion
```

Reply:
Why should this be deleted? Should there not be an explanation of the example?

Reply:
The examples generally only have a brief intro. Sometimes explanations are included but I think this should be reserved for code that is more advanced or ambiguous.

Reply:
I've deleted the explanation.